libraries/core-react/src/components/Popover/PopoverTitle.tsx

```diff
@@ -1,24 +1,22 @@
 import { forwardRef, HTMLAttributes } from 'react'
 import styled from 'styled-components'
 import { Divider } from '../Divider'
-import { typographyTemplate } from '../../utils'
 
 import { popover as tokens } from './Popover.tokens'
 
 const StyledPopoverTitle = styled.div`
-  ${typographyTemplate(tokens.header)}
-  margin-right: ${tokens.closeButton.width};
-  max-width: calc(${tokens.popover.maxWidth} - ${tokens.closeButton.width});
+  margin-right: ${tokens.entities.closeButton.width};
```

Comment:
This looks the wrong use of a token value? Should this not be a `spacings.right` value on `closeButton` instead?

Reply:
This margin is to leave space for the button, had to do this when I moved the button into `Popover` instead of `PopoverTitle` a while ago.